content/applications/hr/lunch.rst

```diff
@@ -0,0 +1,313 @@
+vendors, locations, products, product categories, and alerts. Once these are created, employees can
+view offerings and order food.
+
+Settings
+========
+
+Only 2 settings are needed to configure in the *Lunch* app: overdraft settings and notifications.
+To access the settings, navigate to :menuselection:`Lunch app --> Configuration: Settings`.
+
+Configure the following:
+
+- :guilabel:`Lunch Overdraft`: enter the maximum overdraft amount for employees. The currency
+  format is determined by the localization setting of the company.
+- :guilabel:`Reception notification`: set the message users receive via the *Discuss* app when their
+  food has been delivered. The default message `Your lunch has been delivered. Enjoy your meal!`
+  populates this field, but can be modified, if desired.
+
+.. tip::
+   If in a database with multiple languages installed, many forms in the *Lunch* application have
+   the option of entering translations for various fields.
+
+   If translations are available to be configured, a language code appears next to a translatable
+   field on a form. To add translations for that field, click the two letter language code (for
+   example, :guilabel:`EN` for English) and a translation pop-up window appears.
+
+   The following is an example for the :guilabel:`Reception notification` field in the settings
+   menu:
+
+   Navigate to :menuselection:`Lunch app --> Configuration: Settings`. Click the :guilabel:`EN` In
+   the top-right of the text box beneath the :guilabel:`Reception notification` section. A
+   :guilabel:`Translate: company_lunch_notify_message` pop-up window loads with the option to enter
+   a translation for the other languages used by the database.
+
+   The first column lists the different languages in alphabetical order, with the currently selected
+   language in bold. The second column has the currently configured message in each column. The last
+   column in the far-right provides a text box to type in a translation for each language.
+
+   Enter the text that should appear for each language, then click :guilabel:`Save`.
+
+   Note that for many forms, the :guilabel:`Save` button is not required, and may not appear.
+
+  .. image:: lunch/translation.png
+     :align: center
+     :alt: The translation text box, with the current language highlighted, and the Arabic
+           translation field highlighted.
+
+Vendors
+=======
+
+Before food can be ordered, the restaurants that make the food, referred to as *vendors* in Odoo,
+**must** be configured.
+
+First navigate to :menuselection:`Lunch app --> Configuration: Vendors`, and all currently
+configured vendors for the *Lunch* app appear in a default Kanban view. To change to a list view,
+click the :guilabel:`(list icon)` in the top-left corner.
+
+To add a new vendor, click the :guilabel:`New` button in the top-left corner, and a new lunch
+supplier form loads.
+
+Fill out the following fields on the form:
+
+Vendor Information section
```

Comment:
Sentence style heading, since this isnt a form walkthrough and no UI text stating "Vendor Information" :)
```suggestion
Vendor information section
```